services/consensuscontext/service.go

```diff
@@ -63,7 +63,7 @@ func (s *service) RequestNewTransactionsBlock(ctx context.Context, input *servic
 
 	s.logger.Info("created Transactions block", log.Int("num-transactions", len(txBlock.SignedTransactions)), log.Stringable("transactions-block", txBlock))
 
-	s.metrics.numberOfTransactions.Measure(int64(len(txBlock.SignedTransactions)))
+	s.metrics.transactionsRate.Measure(int64(len(txBlock.SignedTransactions)))
```

Comment:
why do we care about tx rate here? we already measure it in txpool - do you expect a different measurement here?
alternately, and as we discussed - how about a histogram for distributions of block sizes?

Reply:
We can add histogram later when we start to apply pressure tests.